c2pie/signing.py

```diff
@@ -37,9 +88,8 @@ def _load_certificates_and_key(
 
 
 def sign_file(
-    file_type: C2PA_ContentTypes,
-    input_path: str,
-    output_path: str,
+    input_path: Path | str,
+    output_path: Path | str | None = None,
     key_path: str | None = os.getenv("C2PIE_KEY_FILEPATH"),
     certificates_path: str | None = os.getenv("C2PIE_CERT_FILEPATH"),
```

Comment:
```suggestion
    key_path: str | None = None,
    certificates_path: str | None = None,
```

It's better to load them from environment variables inside `_load_certificates_and_key`